module/VuFind/src/VuFind/Auth/LoginTokenManager.php

```diff
@@ -131,23 +139,23 @@ class LoginTokenManager implements \VuFind\I18n\Translator\TranslatorAwareInterf
     /**
      * LoginToken constructor.
      *
-     * @param Config                                   $config          Configuration
-     * @param UserTable                                $userTable       User table gateway
-     * @param LoginTokenTable                          $loginTokenTable Login Token table gateway
-     * @param CookieManager                            $cookieManager   Cookie manager
-     * @param SessionManager                           $sessionManager  Session manager
-     * @param \VuFind\Mailer\Mailer                    $mailer          Mailer
-     * @param \Laminas\View\Renderer\RendererInterface $viewRenderer    View Renderer
-     * @param callable                                 $browscapCB      Callback for creating Browscap
+     * @param Config            $config          Configuration
+     * @param UserTable         $userTable       User table gateway
+     * @param LoginTokenTable   $loginTokenTable Login Token table gateway
+     * @param CookieManager     $cookieManager   Cookie manager
+     * @param SessionManager    $sessionManager  Session manager
+     * @param Mailer            $mailer          Mailer
+     * @param RendererInterface $viewRenderer    View Renderer
+     * @param callable          $browscapCB      Callback for creating Browscap
      */
     public function __construct(
-        \Laminas\Config\Config $config,
-        \VuFind\Db\Table\User $userTable,
-        \VuFind\Db\Table\LoginToken $loginTokenTable,
-        \VuFind\Cookie\CookieManager $cookieManager,
+        Config $config,
+        Usertable $userTable,
```

Comment:
This should be `UserTable`, I believe... and if this typo didn't break a test, it's probably a sign that we're missing an important test.

Reply:
Class names are not case-sensitive (though autoloaders usually are). Fixed anyway.